src/bci_build/package/basecontainers.py

```diff
@@ -173,11 +173,14 @@ def _get_supported_until_fips(os_version: OsVersion) -> datetime.date:
             os_version in CAN_BE_LATEST_OS_VERSION or os_version in ALL_OS_LTSS_VERSIONS
         ),
         pretty_name=_get_fips_pretty_name(os_version),
-        package_list=[*os_version.release_package_names, "coreutils"]
-        + (
-            ["fipscheck"]
-            if os_version == OsVersion.SP3
-            else ["crypto-policies-scripts"]
+        package_list=(
+            [*os_version.release_package_names, "coreutils"]
+            + (
+                ["fipscheck"]
+                if os_version == OsVersion.SP3
+                else ["crypto-policies-scripts"]
+            )
+            + (["patterns-base-fips"] if os_version.is_slfo else [])
```

Comment:
Note that this has no effect atm, as this container is not being built for SLE 16